doc/FEATURES_RELEASE_2.md

```diff
@@ -41,6 +41,9 @@ DDI. Press `Submit`. The system should display the DDI information
 for the DDI found, including whether its disposition is `Good` or `Bad`,
```

Comment:
In the product backlog this feature refers only to CSV generation. Perhaps we should mention two separate things in this section:
1) The terminology he asked for (`Good`/`Bad`) is used in the CSV
2) terminology we display (`Harmful`/`Not harmful`) was chosen because it is more suitable for the use case. `harmful` for the patient may not necessarily be `bad`. Some treatments are `harmful` but overall have a `good` effect.